build/cloud/ha/02-ha-configmap.yaml.example

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloudcore
+  namespace: kubeedge
+  labels:
+    k8s-app: kubeedge
+    kubeedge: cloudcore
+data:
+  cloudcore.yaml: |
+    apiVersion: cloudcore.config.kubeedge.io/v1alpha1
+    kind: CloudCore
+    kubeAPIConfig:
+      kubeConfig: ""
+      master: ""
+    leaderelection:
+      LeaderElect: true
+      LeaseDuration: 15s
+      RenewDeadline: 10s
+      ResourceLock: endpointsleases
+      ResourceName: cloudcorelease
+      ResourceNamespace: kubeedge
+      RetryPeriod: 2s
+    modules:
+      cloudHub:
+        advertiseAddress:
+         - YOUR CLOUDCORE VIP HERE !!!
```

Comment:
I'd like this is something that could be replaced by env.

Reply:
kevin said that `.example` means this file need user to configure mannually. Now I do not have a good idea to auto inject `env` to this file, in fact `env` also need to be seted by user mannully because the `advertiseAddress` is uncertain.

Reply:
Now the example is fine, let's set it in helm chart later